lib/api/web-element/commands/click.js

```diff
@@ -23,5 +23,5 @@
  * @returns {ScopedWebElement}
  */
 module.exports.command = function() {
-  return this.runQueuedCommand('clickElement');
+  return this.runQueuedCommand('clickElement', {isTerminal: true});
```

Comment:
@dikwickley For this, we don't need to introduce any new flag because all terminal commands call `this.runQueuedCommand()` method so we can use that to make our changes.

Reply:
yes, it seems it was redundant. Also I updated the return object to match with what we get from the old API commands (something in lines of)
```
{ error:..., message:....., stack:....}
```
<img width="1185" alt="Screenshot 2024-05-19 at 6 06 40 PM" src="https://github.com/nightwatchjs/nightwatch/assets/31622972/a456ea2c-484d-4a90-a98f-6e1721e1fc63">